spec/models/run_spec.rb

```diff
@@ -589,4 +589,39 @@
       expect(run.total_playtime_ms).to eq 8_589_280
     end
   end
+
+  context 'skipped splits stats' do
+    let(:run) do
+      r = FactoryBot.create(:skipped_splits_run)
+      r.parse_into_db
+      r.reload
+      r
+    end
+
+    it 'has the correct stats' do
+      stats = run.segment_history_stats(Run::REAL)
+      ids = run.segments.order(:segment_number).map(&:id)
+
+      expect(stats[ids[0]][:standard_deviation].to_f).to eq 48989.79485566
```

Comment:
We've had problems in the past with using exact decimals to this precision, might be better to use the `be_within` matcher.